src/pages/Signup/index.jsx

```diff
@@ -190,31 +197,80 @@ const SignupScreen = () => {
     e.preventDefault();
     auth
       .signInWithPopup(googleProvider)
-      .then(() => {
-        playSuccessSound();
-        enqueueSnackbar("Signin successful!", {
-          variant: "success",
-        });
-        navigate("/dummygram");
+      .then(async (val) => {
+        setFullName(val?.user?.displayName);
+        setEmail(val?.user?.email);
+        setIsOauthSignUp(true);
+        const usernameDoc = db.collection(`users`);
```

Comment:
we don't need to store this perhaps, we can call it directly

Reply:
Done!.